integ-test/src/test/java/org/opensearch/sql/sql/MatchPhraseIT.java

```diff
@@ -36,10 +36,29 @@ public void test_matchphrase_legacy_function() throws IOException {
     verifyDataRows(result, rows("quick fox"), rows("quick fox here"));
   }
 
+  @Test
+  public void test_matchphrasequery_legacy_function() throws IOException {
+    String query = "SELECT phrase FROM %s WHERE matchphrasequery(phrase, 'quick fox')";
+    JSONObject result = executeJdbcRequest(String.format(query, TEST_INDEX_PHRASE));
+    verifyDataRows(result, rows("quick fox"), rows("quick fox here"));
+  }
+
   @Test
   public void test_match_phrase_with_slop() throws IOException {
     String query = "SELECT phrase FROM %s WHERE match_phrase(phrase, 'brown fox', slop = 2)";
     JSONObject result = executeJdbcRequest(String.format(query, TEST_INDEX_PHRASE));
     verifyDataRows(result, rows("brown fox"), rows("fox brown"));
   }
+
+  @Test
+  public void test_alternate_syntax_for_match_phrase_returns_same_result() throws IOException {
+    String query1 = "SELECT phrase FROM %s WHERE matchphrase(phrase, 'quick fox')";
+    String query2 = "SELECT phrase FROM %s WHERE match_phrase(phrase, 'quick fox')";
+    String query3 = "SELECT phrase FROM %s WHERE matchphrasequery(phrase, 'quick fox')";
+    JSONObject result1 = executeJdbcRequest(String.format(query1, TEST_INDEX_PHRASE));
+    JSONObject result2 = executeJdbcRequest(String.format(query2, TEST_INDEX_PHRASE));
+    JSONObject result3 = executeJdbcRequest(String.format(query3, TEST_INDEX_PHRASE));
+    assertEquals(result1.toString(), result2.toString());
```

Comment:
`assertEquals(result1, result2)` should work.

Reply:
I tried to just do `assertEquals(result1, result2)`  originally but the test was failing with the following message (I separated it onto different lines so it's more readable)...
```
expected: 
org.json.JSONObject<{"schema":[{"name":"phrase","type":"text"}],"total":2,"datarows":[["quick fox"],["quick fox here"]],"size":2,"status":200}> 
but was: 
org.json.JSONObject<{"schema":[{"name":"phrase","type":"text"}],"total":2,"datarows":[["quick fox"],["quick fox here"]],"size":2,"status":200}>
```

I wasn't sure why it was failing, but I assumed it may have something to do with the equality operator and how the objects might be stored in memory, so I added the `.toString()` as a hacky way to fix it. Is there a different way to compare the JSON objects that might be better?

Reply:
I found that it is better to use `JSONObject::similar` instead of `JSONObject::equals`. So assert be like
```
assertTrue(result1.similar(result2));
```

Reply:
This has been fixed. `assertEquals` was replaced with `assertTrue`.